src/api/spec/components/previews/bs_request_history_element_component_preview.rb

```diff
@@ -1,9 +1,16 @@
 class BsRequestHistoryElementComponentPreview < ViewComponent::Preview
+  # /rails/view_components/bs_request_history_element_component/with_history_element_request_superseded
   def with_history_element_request_superseded
     render(BsRequestHistoryElementComponent.new(element: HistoryElement::RequestSuperseded.last))
   end
 
+  # /rails/view_components/bs_request_history_element_component/with_history_element_request_accepted
   def with_history_element_request_accepted
     render(BsRequestHistoryElementComponent.new(element: HistoryElement::RequestAccepted.last))
   end
+
+  # /rails/view_components/bs_request_history_element_component/with_history_element_request_review_added
+  def with_history_element_request_review_added
+    render(BsRequestHistoryElementComponent.new(element: HistoryElement::RequestReviewAdded.last))
```

Comment:
@saraycp hmm, I'm not sure if its good to depend pre existing database entries for the previews. Sometimes we create the records in the preview itself, sometime we depend on the existence of them in the DB. Anyway you don't introduce this, there are other previews that do the same. But I think I will create a card to start talking about this.

Reply:
card is in the backlog now

Reply:
I totally agree.

Reply:
I created a Free Style [card](https://trello.com/c/wGPrnopV/2860-set-data-for-components-previews).